lemmatization/services/base.py

```diff
@@ -47,6 +47,22 @@ def normalize(self, word):
         """
         return word
 
+    def check_text(self, text):
+        """
+        Returns TRUE or False based on the
```

Comment:
"...rules applied to modify the text", like the next function?

Reply:
Good catch! I was referring to the rule for the text based on specific text(e.g. latin uses and underscore to join specific lemmas together). Not necessarily modifying it for this function but we could combine the two functions in the future.
I'm mainly using this function to see if we should keep a record of the `chunk` to be joined later in the code